html/janus.js

```diff
@@ -1539,30 +1523,26 @@ function Janus(gatewayCallbacks) {
 					config.dtmfSender.ontonechange = function(tone) { Janus.debug("Sent DTMF tone: " + tone.tone); };
 				}
 			}
-			if(config.dtmfSender === null || config.dtmfSender === undefined) {
+			if(!config.dtmfSender) {
 				Janus.warn("Invalid DTMF configuration");
 				callbacks.error("Invalid DTMF configuration");
 				return;
 			}
 		}
 		var dtmf = callbacks.dtmf;
-		if(dtmf === null || dtmf === undefined) {
+		if(!dtmf) {
 			Janus.warn("Invalid DTMF parameters");
 			callbacks.error("Invalid DTMF parameters");
 			return;
 		}
 		var tones = dtmf.tones;
-		if(tones === null || tones === undefined) {
+		if(!tones) {
 			Janus.warn("Invalid DTMF string");
 			callbacks.error("Invalid DTMF string");
 			return;
 		}
-		var duration = dtmf.duration;
-		if(duration === null || duration === undefined)
-			duration = 500;	// We choose 500ms as the default duration for a tone
-		var gap = dtmf.gap;
-		if(gap === null || gap === undefined)
-			gap = 50;	// We choose 50ms as the default gap between tones
+		var duration = dtmf.duration || 500; // We choose 500ms as the default duration for a tone
+		var gap = dtmf.gap || 50; // We choose 50ms as the default gap between tones
```

Comment:
`duration` and `gap` are allowed to be `0`: this change breaks that, and incorrectly sets them to the default value instead when `0` is provided.